package-lock.json

```diff
@@ -348,6 +350,24 @@
 				}
 			}
 		},
+		"@automattic/notifications": {
+			"version": "file:apps/notifications",
+			"requires": {
+				"@automattic/calypso-color-schemes": "file:packages/calypso-color-schemes",
+				"@automattic/calypso-polyfills": "file:packages/calypso-polyfills"
+			}
+		},
+		"@automattic/o2-blocks": {
+			"version": "file:apps/o2-blocks",
+			"requires": {
+				"@wordpress/blocks": "6.12.0",
```

Comment:
We also seem to be missing `@wordpress/api-fetch`, `@wordpress/hooks`,  `@wordpress/block-editor`, `@wordpress/data`  and even `moment` in the package.json file for o2-blocks.

Reply:
That's because these packages don't need to be present in `node_modules`. They are always externalized by a webpack plugin. For a Gutenberg block, `import '@wordpress/...'` statements are not resolved to a NPM module.

There are still reasons to declare them as `dependencies` -- for example to use type definitions published by the package.